queue_services/account-mailer/src/account_mailer/email_templates/pad_invoice_email.html

```diff
@@ -1,4 +1,7 @@
-# Your {{ account_number }}: {{ account_name_with_branch }} account completed ${{ invoice_total }} in transactions during business hours on {{ invoice_process_date }}.
+# This email confirms a recent transaction on you account {{ account_number }}: {{ account_name_with_branch }}.
+
+Transaction date: ${{ invoice_process_date }}
+Invoice reference number: {{ invoice_number }}
```

Comment:
How are you handling the scenario where there are multiple invoice_numbers, are you just taking the first?